content/en/docs/concepts/project-maturity.md

```diff
@@ -6,23 +6,19 @@ type: "docs"
 ---
 
 cert-manager has wide adoption in the Kubernetes community with it being in use
-in both production and non-production clusters. The project is still in an alpha
-state where it has yet to reach `v1.0`.
+in both production and non-production clusters.
 
 ## API
 
-The cert-manager API is currently in an `v1alpha2` version and as such is
-subject to change. We foresee one beta version of `v1beta1` at which we expect
-minimal changes, if any, to the next version release of `v1`. We expect to hit
-version `v1` late 2019, early 2020.
+The cert-manager API is currently at `v1` and is stable.
 
 ## Compatibility
 
 cert-manager has a hard guarantee of compatibly with the current stable upstream
```

Comment:
This is not true any longer due to OpenShift 3.11 being supported for 4 years we somehow are also bound to that. Should be made into an issue to be handled later

Reply:
See https://github.com/cert-manager/website/issues/305